lib/LaTeXML/Package/pgfmath.code.tex.ltxml

```diff
@@ -297,8 +297,19 @@ sub pgfmathparse {
   my $input = $string;
 ###    if ($string =~/^+/){        # Don't parse as expression, just as glue. ???
   #    print STDERR "\nPGF Math Parse: $string\n";
-  $PGFMATHGrammar = Parse::RecDescent->new($PGFMATHGrammarSpec) unless $PGFMATHGrammar;
-  my $result = $PGFMATHGrammar->expr(\$string) || 0.0;
+  my $result;
+  {
+    local $@;
```

Comment:
You're going to want 
```
local $LaTeXML::IGNORE_ERRORS = 1;
```
here, otherwise we're still registering errors. They don't get reported, but if they exceed 100, you get a Fatal! which shows up in the final status!